# Sequencing Data
 
## HiFi Data
Sequenced used for the v0.7 assembly were generated by HPRC and GIAB and are available from [AWS](https://s3-us-west-2.amazonaws.com/human-pangenomics/index.html?prefix=T2T/scratch/HG002/sequencing/hifi/). HiFi Revio data were used in the creation of v0.8, v0.9, and v1.0 and are available from [PacBio](https://downloads.pacbcloud.com/public/revio/2022Q4/) as HG002-rep1, HG002-rep2, and HG002-rep3 and on [AWS](https://s3-us-west-2.amazonaws.com/human-pangenomics/index.html?prefix=T2T/scratch/HG002/sequencing/hifirevio/).

A newly available HiFi Revio dataset for HG002 has been provided by PacBio and is posted on [AWS](https://s3-us-west-2.amazonaws.com/human-pangenomics/index.html?prefix=T2T/HG002/assemblies/polishing/HG002/v1.0/mapping/hifi_revio_pbmay24/).
 
## Oxford Nanopore Data
Nanopore sequencing was performed by HPRC and GIAB. The fastq data used for the v0.7 assembly are available from [AWS](https://s3-us-west-2.amazonaws.com/human-pangenomics/index.html?prefix=T2T/scratch/HG002/sequencing/ont/). The raw fast5 files are available from the original data sources at [HPRC](https://s3-us-west-2.amazonaws.com/human-pangenomics/index.html?prefix=NHGRI_UCSC_panel/HG002/nanopore/), [GIAB](https://ftp-trace.ncbi.nlm.nih.gov/ReferenceSamples/giab/data/AshkenazimTrio/HG002_NA24385_son/Ultralong_OxfordNanopore/), and [ONT](https://labs.epi2me.io/dataindex/) (Sept and Nov 2020 releases). For v0.8, v0.9, and v1.0, we made use of R10 duplex data available from [AWS](https://s3-us-west-2.amazonaws.com/human-pangenomics/index.html?prefix=submissions/0CB931D5-AE0C-4187-8BD8-B3A9C9BFDADE--UCSC_HG002_R1041_Duplex_Dorado/Dorado_v0.1.1/stereo_duplex/). In December, 2023, EPI2ME announced the release of roughly 40x coverage of ultra-long R10 reads with read length N50 of 91kbp and a median accuracy of Q26.4. These reads, often referred to as "Q28", are available from [EPI2ME](https://labs.epi2me.io/gm24385_ncm23_preview/#sample-preparation-and-analysis).

## Element Biosciences Data
Element Biosciences whole genome data from PCR-free libraries for the entire HG002 trio was used in creating v0.9 and v1.0, and is available on [AWS](https://s3-us-west-2.amazonaws.com/human-pangenomics/index.html?prefix=T2T/scratch/HG002/sequencing/element/trio/) and on the [Element Biosciences website](https://go.elementbiosciences.com/human-whole-genome-sequencing-cloudbreak).

## PacBio Onso Sequence Data
Onso sequencing data was provided by Mark Fleharty at the Broad Institute and is available on [AWS](https://s3-us-west-2.amazonaws.com/human-pangenomics/index.html?prefix=T2T/scratch/HG002/sequencing/onso/Broad-Onso-HG002/). Additional Onso sequencing data was provided by Chris Mason at Weill Cornell Medicine and is also available on [AWS](https://s3-us-west-2.amazonaws.com/human-pangenomics/index.html?prefix=T2T/scratch/HG002/sequencing/onso/Mason-Onso-HG002/).

## StrandSeq
Strandseq data is available from the [HPRC](https://s3-us-west-2.amazonaws.com/human-pangenomics/index.html?prefix=working/HPRC_PLUS/HG002/raw_data/Strand_seq/)
 
## HiC
Hi-C data is available from the [HPRC](https://s3-us-west-2.amazonaws.com/human-pangenomics/index.html?prefix=working/HPRC_PLUS/HG002/raw_data/hic/downsampled/). The assembly used HG002.HiC_2_NovaSeq_rep1_run2_S1_L001_R1_001.fastq.gz and HG002.HiC_2_NovaSeq_rep1_run2_S1_L001_R2_001.fastq.gz.
 
## Illumina PCRFree Data
For polishing, we made use of 2x250 whole genome sequence from Illumina available from [NCBI](https://ftp-trace.ncbi.nlm.nih.gov/giab/ftp/data/AshkenazimTrio/HG002_NA24385_son/NIST_Illumina_2x250bps/).

## Other listed resources
A comprehensive catalog of available sequencing data for HG002 is available on the FTP site at [NCBI](https://ftp-trace.ncbi.nlm.nih.gov/ReferenceSamples/giab/data/AshkenazimTrio/HG002_NA24385_son/).

# Notes on downloading files.
 
Files are generously hosted by Amazon Web Services. Although available as straight-forward HTTP links, download performance is improved by using the Amazon Web Services <a href="https://aws.amazon.com/cli/">command-line interface</a>. References should be amended to use the `s3://` addressing scheme, i.e. replace `https://s3-us-west-2.amazonaws.com/human-pangenomics/T2T/` with `s3://human-pangenomics/T2T` to download.
 
The s3 command can also be used to get information on the dataset, for example reporting the size of every file in human-readable format.
 
    aws s3 --no-sign-request ls --recursive --human-readable --summarize s3://human-pangenomics/T2T/scratch/HG002/
 
or to obtain technology-specific sizes.
 
    aws s3 --no-sign-request ls --recursive --human-readable --summarize s3://human-pangenomics/T2T/scratch/HG002/sequencing
    aws s3 --no-sign-request ls --recursive --human-readable --summarize s3://human-pangenomics/T2T/scratch/HG002/assemblies
 
Amending the `max_concurrent_requests` etc. settings as per <a href="https://docs.aws.amazon.com/cli/latest/topic/s3-config.html">this guide</a> will improve download performance further.
 
You can also browse all the files available on S3 via <a href="https://s3-us-west-2.amazonaws.com/human-pangenomics/index.html?prefix=T2T/HG002/assemblies">web interface</a>.

